.github/workflows/run-tests.yml

```diff
@@ -25,7 +25,9 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: 'npm'
-      - name: Install dependencies
+      - name: Install npm dependencies
         run: npm ci
       - name: Run tests
         run: npm test
+      - name: Build static site
+        run: bundle exec middleman build --verbose
```

Comment:
Does the `--verbose` option have any risk of printing out e.g. environment variables/secrets?

Reply:
Not as far as I know. Middleman does not need access to any env variable/secrets to run, nor we pass on any to it.